src/main/rx/rx.c

```diff
@@ -449,6 +449,14 @@ bool rxUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTime)
             rxIsInFailsafeMode = (frameStatus & RX_FRAME_FAILSAFE) != 0;
             rxSignalReceived = !rxIsInFailsafeMode;
             needRxSignalBefore = currentTimeUs + needRxSignalMaxDelayUs;
+
+            if (frameStatus & RX_FRAME_DROPPED) {
+            	// No (0%) signal
+            	setRssiUnfiltered(0, RSSI_SOURCE_NONE);
+            } else {
+            	// Valid (100%) signal
+            	setRssiUnfiltered(1024, RSSI_SOURCE_NONE);
```

Comment:
1024 is outside of the valid range. Maximum is 1023 (`RSSI_MAX_VALUE`).

Reply:
Use the symbolic name available:
`src/main/rx/rx.c:#define RSSI_MAX_VALUE 1023`

Reply:
And now I see this is defined in the `rx.c` file!! Should be moved to a header files, to be reusable. Maybe `rx.h`.

Reply:
I originally used RSSI_MAX_VALUE, but that required the RSSI Maximum setting in the OSD to be reduced from 1023 to 1022 or the max value displayed was 99%. Clearly a rounding error. I'll track that down.

Reply:
Also, note that rx.h already defines RSSI_SCALE_DEFAULT with the comment '100% @ 4095' so we have conflicting ranges. And there's also the definition '#define RSSI_SCALE_MAX 255'. In various places we see the upper limit of RSSI to be 100, 255, 1023 or 4095. I'll see if we can rationalise this without too much issue.